File: Array/Deletion.py
def delete(arr, pos):
    n = len(arr)
    for i in range(pos, n-1):
        arr[i] = arr[i+1]
    arr.pop()


n = int(input('Enter size of Array: '))
arr = list(map(int, input().split()))
pos = int(input('Enter position of item to delete: '))
delete(arr, pos)
print(arr)

File: Array/Deletion.java
package Array;

import java.util.Scanner;

public class Deletion {

  public static void deleteFromArray(int[] arr, int pos) {
    for (int i = pos; i < arr.length - 1; i++) {
      arr[i] = arr[i+1];
    }
  }

  public static void main(String[] args) {

    Scanner scanner = new Scanner(System.in);
    int[] arr = Basic.inputArray();
    System.out.print("Enter position: ");
    int pos = scanner.nextInt();
    deleteFromArray(arr, pos);
    System.out.println("Updated Array is:");
    for (int i = 0; i < arr.length - 1; i++) {
      System.out.print(arr[i] + " ");
    }
    scanner.close();
  }
  
}
